Form-Controls/styles.css

```diff
@@ -0,0 +1,83 @@
+  body {
+    font-family: Arial, sans-serif;
+    background-color: #FFFDF5;
+    line-height: 1.5;
+    min-height: 100vh;
+    display: flex; 
+    flex-direction: column;
+	  margin: 0;
+  }
+  
+  header {
+    text-align: center;
+  }
+
+  h1 {
+    color: #003049;
+  }
+  
+  main {
+    background-color: #ebbab9; 
+    border-radius: 8px;
+    margin: 1rem auto;
+    width: 90%;
+    max-width: 400px;
```

Comment:
Hi @Yulia-I19 ,
Have you considered the benefits of using rem over px? I think px should be used only for elements with fixed sizes (e.g. border, and box-shadow). I think rem is more suitable. 

See link below for more context.
https://www.shecodes.io/athena/28078-when-to-use-ems-rems-and-px-in-css